tasks/install.yml

```diff
@@ -1,13 +1,34 @@
 ---
 
-- import_tasks: user.yml
+- name: Check presence of fastestmirror.conf
+  stat:
+    path: /etc/yum/pluginconf.d/fastestmirror.conf
+  register: fastestmirror
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile:
+    dest: /etc/yum/pluginconf.d/fastestmirror.conf
+    regexp: "^enabled=.*"
+    line: "enabled=0"
+    state: present
+  when: fastestmirror.stat.exists
 
 - import_tasks: postgresql.yml
   when: not patroni_postgresql_exists
 
 - import_tasks: watchdog.yml
   when: patroni_watchdog_mode in ('automatic', 'required')
 
+- name: Install EL for system packages

```

Comment:
Please, use the suggested approach with vars/RedHat.yml file and `patroni_system_packages` variable as mentioned above.

Reply:
Moved patroni_el_packages to RedHat.yml